Sources/REventLogger/Networking/NewtorkManager.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+
+class NetworkManager {
+
+    private let defaultSession: URLSessionProtocol
+    private var dataTask: DataTaskProtocol?
+
+    init(session: URLSessionProtocol = URLSession(configuration: URLSessionConfiguration.ephemeral)) {
```

Comment:
any reason to make this `URLSessionConfiguration.ephemeral`?

Reply:
It will behave as a default session if we dont make any customisations . Also ephhermal session doesnot store in cache .